cmd/gateway-gcs.go

```diff
@@ -188,14 +191,56 @@ func gcsToObjectError(err error, params ...string) error {
 // gcsProjectIDRegex defines a valid gcs project id format
 var gcsProjectIDRegex = regexp.MustCompile("^[a-z][a-z0-9-]{5,29}$")
 
-// isValidGCSProjectId - checks if a given project id is valid or not.
-// Project IDs must start with a lowercase letter and can have lowercase
-// ASCII letters, digits or hyphens. Project IDs must be between 6 and 30 characters.
+// gcsProjectIDValidFormat - checks if a given project id format is valid or not.
```

Comment:
Why did you change the name from `isValidGCSProjectID`? It captures the intent better than `gcsProjectIDValidFormat`. You could call it `isValidGCSProjectIDFormat` to be more precise.